functions-and-operators/string-functions.md

```diff
@@ -115,11 +115,36 @@ Return the character for each integer passed.
 
 ### [`CHAR_LENGTH()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_char-length)
 
-Return number of characters in argument.
+The `CHAR_LENGTH()` function is used to get the number of characters in a specified expression. It returns the total number of characters in a specified expression as an integer value. 
+
+Examples:
+
+```sql
+SELECT CHAR_LENGTH("TiDB") AS LengthOfString;
+
++----------+
+| LengthOfString |
++----------+
+| 4  |
++----------+
+```
+
+```sql
+SELECT CustomerName, CHAR_LENGTH(CustomerName) AS LenghtOfName FROM Customers;
+
+| CustomerName|LengthOfName  |
+|-------------|--------------|
+| Albert Einstein | 15 |
+| Robert Oppenheimer | 18 |
+```
+
+> **Note:**
+>
+> The second example operates under the assumption that there is a database with a record titled `Customers` and a field inside titled `CustomerName`
```

Comment:
Not sure if the word "record" is proper, in my understanding, it's a table named "Customers" with a column named "CustomerName"

Reply:
changed to row and column to ensure consistency